rcsb/workflow/targets/ProteinTargetSequenceExecutionWorkflow.py

```diff
@@ -31,16 +31,30 @@
 
 
 class ProteinTargetSequenceExecutionWorkflow(object):
-    def __init__(self):
-        self.__mockTopPath = None
-        configPath = os.path.join(HERE, "exdb-config-example.yml")
-        configName = "site_info_remote_configuration"
-        self.__cfgOb = ConfigUtil(configPath=configPath, defaultSectionName=configName, mockTopPath=self.__mockTopPath)
-        self.__cachePath = os.path.join(HERE, "CACHE")
+    def __init__(self, **kwargs):
+        """Workflow wrapper  --  Workflow to rebuild and stash "buildable" cache resources.
+
+        Args:
+            configPath (str, optional): path to configuration file (default: exdb-config-example.yml)
+            configName (str, optional): configuration section name (default: site_info_configuration)
+            cachePath (str, optional):  path to cache directory (default: '.')
+            stashRemotePrefix (str, optional): file name prefix (channel) applied to remote stash file artifacts (default: None)
+        """
+        configPath = kwargs.get("configPath", "exdb-config-example.yml")
+        self.__configName = kwargs.get("configName", "site_info_configuration")
+        mockTopPath = kwargs.get("mockTopPath", None)
+        self.__cfgOb = ConfigUtil(configPath=configPath, defaultSectionName=self.__configName, mockTopPath=mockTopPath)
+        #
```

Comment:
So it looks like you're removing the use of the `HERE` parameter. I thought using that was consistent throughout other Workflow classes, or is it not?

Reply:
I don't think so. I did a quick search and it's in `PubChemExecutionWorkflow` but not most of the others, e.g. those in https://github.com/rcsb/py-rcsb_exdb/tree/master/rcsb/exdb/wf or https://github.com/rcsb/py-rcsb_workflow/tree/master/rcsb/workflow/chem .

Reply:
Hm OK. I still feel it would be good to keep that as the default though (i.e., if you don't specify a working path, then use the directory of the file a.k.a. "HERE"). You may need to add a new parameter `workPath` for that (which you'll find all over in other packages).

Reply:
Can you give me an example of a file that you think has a good configuration to use as a guide? I made the changes to match those files I linked to above, so we should probably update those other ones too if this isn't right.

Reply:
Just pushed these suggested updates. Thanks!